content/docs/started/workstation/getting-started-linux.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Kubeflow on Linux"
+description = "Install Kubeflow on Linux"
+weight = 2
++++
+
+<!--
+  TODO: Create a table that summarizes the options below, helping the user choose
+  more quickly
+-->
+
+## Introduction
+
+For Linux systems you have options for servers and desktops. The server options apply to
+the desktop as well.
+
+## Linux Server
```

Comment:
I think it would make sense to have the desktop section before the server section. Most of the users would like to install Kubeflow on a laptop.

Reply:
In general, I agree - ie I think the Desktop could be more prevalent. The counter-argument is that a developer uses a VM in the cloud for this, particularly for web-based interaction (ie jupyter).  I don't have data to substantiate which will be more popular. The only reason I put the server section up top is because the section is small. An alternative is to describe this up top and provide a link. What do you think?